files/build_templates/docker_image_ctl.j2

```diff
@@ -82,14 +82,14 @@ function preStartAction()
     WARM_DIR=/host/warmboot
     if [[ ("$BOOT_TYPE" == "warm" || "$BOOT_TYPE" == "fastfast")  && -f $WARM_DIR/dump.rdb ]]; then
         # Load redis content from /host/warmboot/dump.rdb
-        docker cp $WARM_DIR/dump.rdb database:/var/lib/redis/dump.rdb
+        docker cp $WARM_DIR/dump.rdb database$DEV:/var/lib/redis/dump.rdb
     else
         # Create an emtpy file and overwrite any RDB if already there
         echo -n > /tmp/dump.rdb
-        docker cp /tmp/dump.rdb database:/var/lib/redis/
+        docker cp /tmp/dump.rdb database$DEV:/var/lib/redis/
     fi
 {%- elif docker_container_name == "snmp" %}
-    sonic-db-cli STATE_DB HSET 'DEVICE_METADATA|localhost' chassis_serial_number $(decode-syseeprom -s)
+    /usr/bin/redis-cli $DEV STATE_DB HSET 'DEVICE_METADATA|localhost' chassis_serial_number $(decode-syseeprom -s)
```

Comment:
>/usr/bin/redis-cli [](start = 4, length = 18)

Can we still using sonic-db-cli? It is for multi-DB feature.

Reply:
Fixed, added a new wrapper sonic-db-cli to invoke sonic-db-cli in specific namespace